base/shell/progman/new_resources.rc

```diff
@@ -56,12 +56,12 @@ CAPTION "New Object"
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
     GROUPBOX        "",IDC_STATIC,5,5,165,44
-    CONTROL         "Program &Group",PM_NEW_GROUP,"Button",BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP,10,4,66,11
-    CONTROL         "&Personal",PM_PERSONAL_GROUP,"Button",BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP,20,15,44,14
-    CONTROL         "&Common",PM_COMMON_GROUP,"Button",BS_AUTORADIOBUTTON,73,15,39,14
+    AUTORADIOBUTTON "Program &Group",PM_NEW_GROUP,10,4,66,11,WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "&Personal",PM_PERSONAL_GROUP,20,15,44,14,WS_GROUP | WS_TABSTOP
```

Comment:
Missing space between the comma and WS-styles here and in the other changes.

Reply:
No, I don't accept that critics. I just adapted to the style that is used throughout this file. There were none before, so there are no also none after. Just check the whole dlg and other dlgs! They do all have no space there.

Reply:
ok thanks for the explanation.